nautobot/core/templates/inc/media.html

```diff
@@ -26,6 +26,9 @@
     <link rel="stylesheet" id="base-theme"
           href="{% static 'css/base.css' %}?v{{ settings.VERSION }}"
           onerror="window.location='{% url 'media_failure' %}?filename=css/base.css'">
+    <link rel="stylesheet" id="base-theme"
```

Comment:
Shouldn't have two elements with the same `id`. But it does make me wonder - should `style.css` be merged into `base.css` rather than creating a new file?